lib/octokit/client/reactions.rb

```diff
@@ -0,0 +1,154 @@
+module Octokit
+  class Client
+
+    # Methods for the Reacions API
+    #
+    # @see https://developer.github.com/v3/reactions/
+    module Reactions
+
+      # List reactions for a commit comment
+      #
+      # @param repo [String, Hash, Repository] A GitHub repository
+      # @param id [Integer] The id of the commit comment
+      # @see https://developer.github.com/v3/reactions/#list-reactions-for-a-commit-comment
+      #
+      # @example
+      #   @client.commit_comment_reactions("octokit/octokit.rb", 1)
+      #
+      # @return [Array<Sawyer::Resource>] Array of Hashes representing the reactions.
+      def commit_comment_reactions(repo, id, options = {})
+        options = ensure_api_media_type(:reactions, options)
+        get "#{Repository.path repo}/comments/#{id}/reactions", options
+      end
+
+      # Create a reaction for a commit comment
+      #
+      # @param repo [String, Hash, Repository] A GitHub repository
+      # @param id [Integer] The id of the commit comment
+      # @param reaction [Integer] The Reaction
+      # @see https://developer.github.com/v3/reactions/#create-reaction-for-a-commit-comment
```

Comment:
Since the api doesn't have a method yet for listing the possible reactions. We should add a `@see` link to https://developer.github.com/v3/reactions/#reaction-types